components/eamxx/src/physics/nudging/atmosphere_nudging.cpp

```diff
@@ -40,9 +38,21 @@ void Nudging::set_grids(const std::shared_ptr<const GridsManager> grids_manager)
   scorpio::register_file(datafile,scorpio::Read);
   m_num_src_levs = scorpio::get_dimlen_c2f(datafile.c_str(),"lev");
   double time_value_0= scorpio::read_time_at_index_c2f(datafile.c_str(),0);
-  std::cout<<"time_value_0: "<<time_value_0<<std::endl;
   double time_value_1= scorpio::read_time_at_index_c2f(datafile.c_str(),1);
-  std::cout<<"time_value_1: "<<time_value_1<<std::endl;
+  double time_value_2= scorpio::read_time_at_index_c2f(datafile.c_str(),2);
+
+  //The times for time_steps are in units of day, not seconds
+  if (time_value_1>0){
+    time_steps.push_back(time_value_1);
+    int n_time_steps = time_value_0/time_value_1;
```

Comment:
I don't get this line. Why would the ratio t1/t0 be a number of time steps?

Reply:
It is because t0 is actually the last time in the file. This is how scorpio::read_time_at_index_c2f was implemented that scorpio::read_time_at_index_c2f(filename,0) will give you the last time in the file. However, this code is actually unnecessary and I removed this block (8 lines from L44-L51). It isn't necessary because I store the time information in the NudgingData struct and compare to the current time.